### ISSUE TITLE
printNewline() misplaced

### ISSUE THREAD

**Issue by DmitryEfimenko** [jbblanchet/jasmine-terminal-reporter#10]
It looks like `formatter.printNewline();` is generally used before the actual message: `formatter.print(message)`. Example:
```javascript
this.suiteStarted = function (suite) {
	if (options.isVerbose) {
		formatter.printNewline();
		formatter.print(formatter.indent(suite.description, verboseIndent));
		verboseIndent += 2;
	}
};
//...
this.specDone = function (result) {
	specCount++;
	var text;
	if (options.isVerbose) {
		formatter.printNewline();
	}
	// the rest of the code
};
//...
this.suiteStarted = function (suite) {
	if (options.isVerbose) {
		formatter.printNewline();
		formatter.print(formatter.indent(suite.description, verboseIndent));
		verboseIndent += 2;
	}
};
```
This is counter-intuitive (usually you first make a statement and then terminate it with a period/separator...) and is a reason for the bug described below.

The bug is that if you have a `beforeAll()` function in your spec, and inside of it there is a `console.log(message)`, that message will appear appended to the suite name on the same line without any spacing, which is not readable. Example:
```javascript
describe('MyTest', ()=> {
    beforeAll(()=> {
        console.log('My message');
    })

    it('should be equal', ()=> {
        // assertitions
    })
});
```
This will print out the following in the console:
> Running 1 spec
> MyTestMy message

It can be easily fixed and I can submit PR if you give me a good to go

**Comment by jbblanchet** [comment]
Thanks for the suggestion. It does make sense, my only concern being that I won't ever be able to support every possible scenario of people outputting to the terminal in the middle of the report.

But in your case I don't see any downside. I'll consider a pull request.

**Comment by DmitryEfimenko** [comment]
At least it'll assure that whatever person outputs is going to be on the new line.
I'll submit PR in a few minutes.

**Comment by jbblanchet** [comment]
Fixed in 1.0.2. Thanks for the PR.
